src/1_utils.js

```diff
@@ -350,20 +350,19 @@ utils.cleanLinkData = function(linkData) {
 		data['$canonical_url'] = utils.getWindowLocation();
 	}
 	if (!data['$og_title']) {
-		data['$og_title'] = utils.getOpenGraphContent('title');
+		data['$og_title'] = utils.setOgTags(data, 'title');
```

Comment:
we can avoid evaluating the condition with every setOgTags call.

`var hasOGRedirectwithFallback = linkData['$og_redirect'] && linkData['$fallback_url'] && linkData['$desktop_url'];`

and use it

`data['$og_description'] = hasOGRedirectwithFallback ? utils.getOpenGraphContent('description') : null;`

And remove the new setOgTags method.

Reply:
Thanks for your advice, I just changed the code based on your feedback.